### DISCUSSION TITLE
Don't Compare Comments in Code Page

### DISCUSSION THREAD

**Discussion by F0RTYS3V3N** [WinMerge/winmerge#1410] [Q&A]
Don't Compare Comments in Code Page

![Adsız](https://user-images.githubusercontent.com/77239430/179830022-1a3b1b45-bebd-4694-b25e-820a036b72f4.png)

example:  " //created by "

**Comment by sdottaka** [comment]
WinMerge has the ability to ignore differences in comments. If you have different comments on the left side of the file, as shown below, you can ignore them. Unfortunately, this does not work in your case because there are no comments on the left side.

![image](https://user-images.githubusercontent.com/98126/179867636-e16e22db-76ab-49bf-8c07-7b80b162e226.png)
